src/main/typescript/components/form/DepositForm.tsx

```diff
@@ -81,17 +82,41 @@ const DepositForm = (props: DepositFormProps) => {
     const formState = useSelector(state => state.depositForm)
     const fileState = useSelector(state => state.files.loading)
     const fileUploadInProgress = useSelector(isFileUploading)
-    const formValues = useSelector(state => state.form.depositForm && state.form.depositForm.values)
+    const formValues = useSelector(state => state.form.depositForm?.values)
     const dispatch = useDispatch()
 
     const doSave = () => {
         const shouldSetToDraft = props.depositState.label === DepositStateLabel.REJECTED
         formValues && dispatch(saveDraft(props.depositId, formValues, shouldSetToDraft))
     }
+    const doAutoSave = () => {
+        // TODO what about REJECTED deposits?
```

Comment:
@jo-pol suggested to do the state change in auto-save for a `REJECTED` deposit as well (element of least surprise).